services-js/registry-certs/client/common/checkout/OrderConfirmationContent.tsx

```diff
@@ -64,11 +68,14 @@ export default function BirthConfirmationContent({
 
           <p className="t--info" style={{ fontStyle: 'normal' }}>
             Have any questions? Email the Registry Department at{' '}
-            <a href="mailto:birth@boston.gov">birth@boston.gov</a>.
+            <a href={`mailto:${certificateType}@boston.gov`}>
```

Comment:
I wonder if you want a map somewhere that’s certificateType -> { email, route, &c. }

Reply:
Good thought, thanks! I’ll merge this PR now and revisit this soon.